### PR TITLE
added getRpcInfo rpc

### PR DESCRIPTION
**Pull request by aikchun**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
Implemented RPC for #48

1. server.getrpcinfo

#### Which issue(s) does this PR fixes?:

Fixes #

### CODE REVIEW

File: packages/jellyfish-api-core/src/category/server.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+export class Server {
+  private readonly client: ApiClient
+
+  constructor (client: ApiClient) {
+    this.client = client
+  }
+
+  /**
+   * Returns details of the RPC server
+   * @return {Promise<RpcInfo>}
```
Comment (jingyi2811) [inline packages/jellyfish-api-core/src/category/server.ts:12]: Add a break line

File: packages/jellyfish-api-core/__tests__/category/server/getRpcInfo.test.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should getRpcInfo', async () => {
+    await waitForExpect(async () => {
+      const info = await client.server.getRpcInfo()
+      expect(info.active_commands.length).toBeGreaterThan(0)
```
[the remark this answers is not in the document]
Comment (fuxingloh) [reply packages/jellyfish-api-core/__tests__/category/server/getRpcInfo.test.ts:42]: @aikchun to assert type is a string for regression purpose, as javascript is not type safe.

File: packages/jellyfish-api-core/__tests__/category/server/getRpcInfo.test.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    await container.start()
+    await container.waitForReady()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should getRpcInfo', async () => {
+    await waitForExpect(async () => {
```
Comment (fuxingloh) [inline packages/jellyfish-api-core/__tests__/category/server/getRpcInfo.test.ts:19]: Are there any reasons for using `waitForExpect`? In the past, there were flaky conditions that require that, it's mostly resolved now. You shouldn't need to use this, it creates non-determinstic tests.

Comment (aikchun) [reply packages/jellyfish-api-core/__tests__/category/server/getRpcInfo.test.ts:19]: > In the past, there were flaky conditions that require that, it's mostly resolved now.

Understood. I know that `waitForExpect` to meant to be used for asynchronous api calls or e2e tests therefore it seems appropriate to be included here. I suppose regular async await will suffice.

**Review by fuxingloh — APPROVED** [review]

File: packages/jellyfish-api-core/__tests__/category/server/getRpcInfo.test.ts (hunk 3 above)
Comment (fuxingloh) [reply packages/jellyfish-api-core/__tests__/category/server/getRpcInfo.test.ts]: For some context that might be relevant for you, the MasterNodeContainer automatically generate blocks to move the ledger forward periodically. We don't use `waitForExpect` as we want to deterministically reach a block at a height. 

E.g. send balance, use `container.generate(1)` to force a block generation to move balance. Then lastly check updated balances.

**Review by canonbrother — APPROVED** [review]

**Pull request merged**